.github/dependabot.yml

```diff
@@ -14,5 +14,13 @@ updates:
   versioning-strategy: lockfile-only
   labels:
   - dependabot-deps-update
+- package-ecosystem: pip
+  directory: /docs
```

Comment:
Also, this separate section shouldn't be necessary, as the previous entry covers the `/docs` dir as well. Here's proof: https://github.com/ansible/pylibssh/pull/457 — a PR updating the constraints file for docs.